ext/dapr-ext-fastapi/setup.cfg

```diff
@@ -5,7 +5,7 @@ author_email = daprweb@microsoft.com
 license = MIT
 license_file = LICENSE
 classifiers =
-    Development Status :: 2 - Pre-Alpha
+    Development Status :: 5 - Production/Stable
```

Comment:
Did we miss this in the 1.0 release?

Reply:
We did, but I'm not sure it is worth cutting a point release over? Unless there is a way to update the pypi metadata without cutting a new release...